src/core/Table/TableCell.tsx

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Cell, CellProps, TableInstance } from 'react-table';
+import cx from 'classnames';
+import { getCellStyle } from './utils';
+import EditableCell, { OnCellEditCallbackParams } from './EditableCell';
+import SubRowExpander from './SubRowExpander';
```

Comment:
Forgot to mention in previous PR but probably should export ES module for `SubRowExpander` instead of using default export.

Reply:
Added